File: Lógica/Lista2/ExPySalaPython/ex1.py
nota1 = float(input("Digite a nota 1: "))
nota2 = float(input("Digite a nota 2: "))
nota3 = float(input("Digite a nota 3: "))

media = (nota1 + nota2 + nota3) / 3

if media >= 6:
    print(f"Aprovado. Média: {media:.2f} ") 
else:
    print(f"Reprovado. Média: {media:.2f}")

File: Lógica/Lista2/ExPySalaC/ex1.c
#include <stdio.h>

void main()
{   
    float nota1,nota2,nota3,media;
    
    printf("Digite a nota 1: ");
    scanf("%f",&nota1);
    
    printf("Digite a nota 2: ");
    scanf("%f",&nota2);
    
    printf("Digite a nota 3: ");
    scanf("%f",&nota3);
    
    media = (nota1 + nota2 + nota3) / 3;
    
    if (media >= 6){
        printf("Aprovado. Média: %.2f",media);
    }
    else{
        printf("Reprovado. Média: %.2f", media);
    }
    
    //return 0;
}